scripts/release-notes.py

```diff
@@ -99,6 +107,28 @@ def main(token, repo, num_commits, exclude_dependabot):
 
     if skipped_dependabot:
         print(f"\n(Skipped {skipped_dependabot} dependabot commit{'' if skipped_dependabot == 1 else 's'})")
+     
+    # Print PRs that don't have changelog label 
+    github = Github(token)
+    repository = github.get_repo(f'jaegertracing/{repo}')
+    changelog_prs = []
+    other_prs = []
+    
+    for pr in repository.get_pulls(state='closed'):
```

Comment:
this will load all historical PRs, but we only need those which are in the commit history since the last releast

Reply:
basically in L91 we already determine PRs